core/derive/tests/compile_tests.rs

```diff
@@ -1,22 +0,0 @@
-// Copyright 2018-2019 Parity Technologies (UK) Ltd.
-//
-// Licensed under the Apache License, Version 2.0 (the "License");
-// you may not use this file except in compliance with the License.
-// You may obtain a copy of the License at
-//
-//     http://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
-
-#[test]
-fn compile_tests() {
-    let t = trybuild::TestCases::new();
-    t.pass("tests/ui/pass/flush.rs");
-    t.pass("tests/ui/pass/allocate_using.rs");
-    t.compile_fail("tests/ui/fail/flush_empty_enum.rs");
-    t.compile_fail("tests/ui/fail/allocate_using_enum.rs");
-}
```

Comment:
Not sure why you would delete all of our tests.

Reply:
Troubleshooting, they'll be back